### PR TITLE
Remove sys.version_info checks

### PR DESCRIPTION
**Pull request by aaronayres35**
part of #987 

This PR removes conditional blocks specific to older, no longer supported, versions of python

### CODE REVIEW

File: tvtk/tests/test_tvtk.py [hunk 1]
```diff
@@ -337,19 +330,14 @@ def test_property(self):
         self.assertEqual(p.color, (0.5, 0.5, 0.5))
 
     def test_points_lookup(self):
-        """ Test if points can be looked up with both int and long keys.
-            Fixes GH Issue 173.
+        """ Test if points can be looked up.
```
Comment (rahulporuri) [inline tvtk/tests/test_tvtk.py:333]:
```suggestion
        """ Test if points can be looked up. Fixes GH Issue 173.
```

**Review by rahulporuri — APPROVED** [review]
LGTM

File: tvtk/tests/test_tvtk.py (hunk 1 above)
Comment (aaronayres35) [reply tvtk/tests/test_tvtk.py:333]: I actually removed the mention of 173 on purpose as the test is now not really related to the issue:
https://github.com/enthought/mayavi/issues/173

ie the test has nothing to do with `long` anymore.

I am happy to leave the comment though if you think that makes more sense

Comment (rahulporuri) [reply tvtk/tests/test_tvtk.py:333]: ahh. thanks for the context. resolving.

**Pull request merged**